### PR TITLE
Fix query string for sites={id}

### PR DESCRIPTION
**Pull request by jaisgit**

### CODE REVIEW

**Comment by tlovett1** [comment]
Can we cast to int before passing it along?

File: classes/class-ep-api.php [hunk 1]
```diff
@@ -142,7 +142,7 @@ public function query( $args, $scope = 'current' ) {
 
 		if ( 'all' === $scope ) {
 			$index = ep_get_network_alias();
-		} elseif ( is_numeric( $scope ) ) {
+		} elseif ( is_int( (int) $scope ) ) {
 			$index = ep_get_index_name( $scope );
```
Comment (tlovett1) [inline classes/class-ep-api.php]: Here is what I meant:

```php
} elseif ( is_numeric( $scope ) ) {
    $index = ep_get_index_name( (int) $scope );
}
```

Comment (jaisgit) [inline classes/class-ep-api.php]: My thought was to catch it in filter_posts_request() as this is likely to be the only time the argument would be passed as a string. But your suggestion works as well.

**Comment by tlovett1** [comment]
Can you rebase this PR? This should just be one simple commit. Sorry!

**Pull request closed without merging**